data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/GrokLogstashPluginAttributesMapper.java

```diff
@@ -84,4 +95,18 @@
         });
         return dataPrepperGrokMatch;
     }
+
+    private void fixNamedCaptures(final Map<String, List<String>> match, final Map<String, String> patternDefinitions) {
+        for (Map.Entry<String, List<String>> entry : match.entrySet()) {
+            final List<GrokNamedCapturesUtil.GrokNamedCapturesPair> grokNamedCapturesPairs = entry.getValue()
+                    .stream().map(GrokNamedCapturesUtil::convertRegexNamedCapturesToGrokPatternDefinitions)
+                    .collect(Collectors.toList());
+            grokNamedCapturesPairs.forEach(grokNamedCapturesPair -> {
+                patternDefinitions.putAll(grokNamedCapturesPair.getMappedPatternDefinitions());
+            });
+            final List<String> fixedNameCapturesPatterns = grokNamedCapturesPairs.stream().map(
```

Comment:
Stylistically, I find it easier to read stream lines where each component of the stream gets its own line. Here, we have a `map` split on two lines and shared with a `collect`.

```
final List<String> fixedNameCapturesPatterns = grokNamedCapturesPairs.stream()
  .map(GrokNamedCapturesUtil.GrokNamedCapturesPair::getMappedRegex)
  .collect(Collectors.toList());
```

It's not a blocker though.